user-service/src/app.service.ts

```diff
@@ -1,8 +1,168 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Logger, OnModuleInit } from '@nestjs/common';
+import {
+  ProfileUpdateEventMsg,
+  ProfileUpdateEventResponse,
+  UserLoginEventMsg,
+  UserLoginEventResponse,
+  UserSignUpEventMsg,
+  UserSignUpEventResponse,
+} from '@lib/src/user/event-msg.dto';
+import { PrismaClient } from '@prisma/client';
+import { JwtService } from '@nestjs/jwt';
+// import { getSnippetCode } from '@lib/One';
+// import { SampleDto } from '@lib/Test.dto';
 
 @Injectable()
-export class AppService {
-  getHello(): string {
-    return 'Hello World!';
+export class AppService extends PrismaClient implements OnModuleInit {
+  constructor(private jwtService: JwtService) {
+    super();
+  }
+  logger = new Logger('User Service');
+  onModuleInit() {
+    this.$connect();
+    this.logger.log('Connected to the database');
+  }
+
+  async getUser(userId: number): Promise<any> {
+    try {
+      const result = await this.user.findUnique({
+        where: {
+          userId,
+        },
+      });
+      if (result) {
+        return Object.keys(result).reduce((acc, key) => {
+          if (key !== 'password') {
+            acc[key] = result[key];
+          }
+          return acc;
```

Comment:
Can you use same model for response, you can look in chef service.

Reply:
ok i will check it